controllers/add_pvcs.go

```diff
@@ -66,6 +66,7 @@ func (a addPVCs) reconcile(ctx context.Context, r *FoundationDBClusterReconciler
 
 			owner := internal.BuildOwnerReference(cluster.TypeMeta, cluster.ObjectMeta)
 			pvc.ObjectMeta.OwnerReferences = owner
+			log.Info("Creating PVC", "name", pvc.Name)
```

Comment:
```suggestion
			log.V(1).Info("Creating PVC", "name", pvc.Name)
```

That should be a debug statement not an info statement. I would prefer if we also log the corresponding namespace and cluster name.